src/Components/People.js

```diff
@@ -0,0 +1,134 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import "./People.css";
+
+const People = () => {
+  const [userInput, setUserInput] = useState("");
+  const [name, setName] = useState("");
+  const [age, setAge] = useState("");
+  const [gender, setGender] = useState("");
+  // const [people, setPeople] = useState("");
+
+  const fetchPeople = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/people");
+      // debugger;
+      res.data.forEach((person) => {
+        if (person.name === userInput) {
+          setAge(person.age);
+          setGender(person.gender);
+          console.log(userInput, name, age);
+        } else if (userInput !== person.name) {
+          setName("Not Found")
+        }
+      });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const handleChange = (e) => {
+    setUserInput(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+   
+    if (userInput === "" ) {
+      setName("Not Found")
+      setAge("")
+      setGender("")
+
+    } else {
+      setName(userInput)
+      fetchPeople();
```

Comment:
This should be called on component did mount, which I believe is the route of you failing the cypress test